packages/editor/src/components/post-actions/actions.js

```diff
@@ -583,6 +583,7 @@ const viewPostAction = {
 
 const postRevisionsAction = {
 	id: 'view-post-revisions',
+	context: 'list',
```

Comment:
I know we only support one context right now, but I expect this will grow and we may endup with more contexts. Would it make sense for context to be array with all the supported contexts?

Reply:
Nothing is public right now, so I think we can add it when we'll need it.